config/routes.rb

```diff
@@ -21,10 +21,16 @@
     end
   end
 
+  concern :statusable do
```

Comment:
I had always conceptually thought of `show` as something that an admin may do from the admin/mediation pages (although there isn't anything in the mocks about doing that) so when we said we want a status page I thought of it as something different (and easily controllable in this way to not have status pages for particular kinds of requests).  I could honestly go either way at this point though.